File: java-interest-project/src/main/java/calculation/Food.java
package calculation;

public class Food {
	
	private double price;
	private final double taxValue;
	private final String name;
	public static final double tax = 0.14;
	public Food(String name, double price)
	{
		this.price = price;
		this.name = name;
		this.taxValue = price * tax;
	}

	public String getName() {
		return name;
	}

	public double getPrice() {
		return price;
	}
	
	public void setPrice(double price){
		this.price = price;
	}

	public double getTaxPrice() {
		return taxValue;
	}
}

File: dotnet-interest-project/dotnet-interest-project/Models/Food.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace dotnet_interest_project.Models
{
    public class Food
    {
        public double Price { get; set; }
        public double TaxValue { get; set; }
        public String Name { get; set; }
        public const double Tax = 0.14;

        public Food(String name, double price)
        {
            this.Price = price;
            this.Name = name;
            this.TaxValue = price * Tax;
        }
    }
}